test/sol5/kyberNetworkProxy.js

```diff
@@ -578,6 +583,237 @@ contract('KyberNetworkProxy', function(accounts) {
             });
         }
     });
+    
+    describe("test enable", async () => {
```

Comment:
better name would be:
 test reading simple values

and add to this describe reading of:

 IKyberNetwork public kyberNetwork;
    IKyberHint public hintHandler; // hint handler pointer for users.

Reply:
change as request